### DISCUSSION TITLE
Previewer does not render when Markdown includes linked images

### DISCUSSION THREAD

**Discussion by fireundubh** [requarks/wiki#3713] [Error / Bug Report]
When an image is linked with Markdown syntax, the previewer does not or cannot render.

The previewer will render if the user scrolls beyond the point where the linked image would be displayed. If the user scrolls back to the point where the linked image would be displayed, the previewer will not render again.

From the user's point of view, while scrolling up and down, the previewer is constantly disappearing and reappearing.

I watched the HTML source in Developer Tools while reproducing the issue and I did not see any dynamic changes.

# Affecting Syntax

This is an example of how images can be linked with Markdown.

```markdown
[![Requarks Logo](https://www.requarks.io/svg/logo.svg)](https://https://www.requarks.io/svg/logo.svg)
```

# Configuration

Component | Flavor/Version
:--- | :---
OS | Ubuntu 17.10
Wiki.js | 2.0.12
DB | MariaDB 10.2.19

**Comment by isiah-lloyd** [comment]
I was not able to reproduce this bug using the latest development version.

**Comment by RED-CAD-Solutions-AG-Development** [comment]
Hello 

I had the same bug or problem with the preview of my markdown text. 

The Preview scrolls automatically to the top. 
![WikiJS_MD_Preview_Bug_](https://user-images.githubusercontent.com/51949837/174999152-ca092576-148c-42c2-bb1c-3990c699196d.gif)

I don't know if it's a syntax failure of me, the structure of the text or something else. 

This is a example to reproduce the bug. 
[Example.txt](https://github.com/requarks/wiki/files/8956508/Example.txt)

**Comment by jcdufourd** [comment]
I had the same problem of images not appearing, and solved it by changing the "Path" of the "Local File System" in the "Storage" section of the admin area from **_relative_** to **_absolute_** (as required in the comment below the field)
